dnf5/main.cpp

```diff
@@ -616,6 +616,23 @@ void RootCommand::set_argument_parser() {
         global_options_group->register_argument(forcearch);
     }
 
+    {
+        auto store = parser.add_new_named_arg("store");
+        store->set_long_name("store");
+        store->set_has_value(true);
+        store->set_arg_value_help("STORED_TRANSACTION_PATH");
+        store->set_description("Store the current transaction.");
+        store->set_parse_hook_func([&ctx](
+                                       [[maybe_unused]] ArgumentParser::NamedArg * arg,
+                                       [[maybe_unused]] const char * option,
+                                       const char * value) {
+            libdnf5::OptionString transaction_store_path(value);
+            ctx.transaction_store_path = transaction_store_path.get_value_string();
+            return true;
+        });
+        global_options_group->register_argument(store);
```

Comment:
Should `--store` be a global argument? It may be better to implement it in `shared_options.cpp` and only register it on the commands that create transactions (`install`, `upgrade`, ... but not `repoquery`, `list`).